components/loader/src/circular-loader/__tests__/circular-loader.test.js

```diff
@@ -0,0 +1,26 @@
+import { mount } from 'enzyme'
+import React from 'react'
+import { CircularLoader } from '../circular-loader.js'
+
+describe('Circular Loader', () => {
+    it('renders the circular loader with aria label', () => {
+        const wrapper = mount(
+            <CircularLoader
+                dataTest={'circular-loader-test'}
+                ariaLabel="Circular Loader"
```

Comment:
the aria-label should actually not be camelCased .. just `aria-label`. Here is a snippet from the docs:

```
In React, all DOM properties and attributes (including event handlers) should be camelCased. For example, the HTML attribute tabindex corresponds to the attribute tabIndex in React. The exception is aria-* and data-* attributes, which should be lowercased. For example, you can keep aria-label as aria-label.
```
https://legacy.reactjs.org/docs/dom-elements.html

Reply:
Noted. Will update

Reply:
@kabaros the props argument for the label is the one that is camelCased. The implementation itself it as `aria-label`. For example: https://github.com/dhis2/ui/blob/ac2b17b5cdbf8941f520a15e7c97a08b554ba6ce/components/loader/src/circular-loader/circular-loader.js#L24 
Should I still make the change?